spec/features/audit/report/allocation_spec.rb

```diff
@@ -1,31 +1,38 @@
 RSpec.feature "Content Allocation", type: :feature do
-  let!(:my_organisation) do
-    create(
-      :organisation,
-    )
-  end
-
-  let!(:me) do
-    create(
-      :user,
-      organisation: my_organisation,
-    )
+  scenario "Filter allocated content" do
+    given_i_am_an_auditor_belonging_to_an_organisation
+    when_i_visit_the_report_page
+    then_i_can_see_the_content_item_i_have_been_allocated
+    when_i_select_no_one_in_the_allocated_to_filter
+    then_i_cannot_see_the_content_item_i_have_been_allocated
   end
 
-  scenario "Filter allocated content" do
-    create(
+  def given_i_am_an_auditor_belonging_to_an_organisation
+    organisation = create(:organisation)
+    user = create(:user, organisation: organisation)
+    @content_item = create(
       :content_item,
       title: "Do Androids Dream of Electric Sheep",
-      allocated_to: me,
-      primary_publishing_organisation: my_organisation,
+      allocated_to: user,
+      primary_publishing_organisation: organisation,
     )
+  end
 
-    visit audits_report_path
+  def when_i_visit_the_report_page
+    @audit_report_page = ContentAuditTool.new.audit_report_page
+    @audit_report_page.load(content_id: @content_item.content_id)
+  end
 
-    expect(page).to have_selector(".report-section", text: "1")
+  def then_i_can_see_the_content_item_i_have_been_allocated
+    expect(@audit_report_page).to have_content_item_count(1)
+  end
+
+  def when_i_select_no_one_in_the_allocated_to_filter
+    @audit_report_page.allocated_to.select 'No one'
+    @audit_report_page.apply_filters.click
+  end
 
-    select "No one", from: "allocated_to"
-    click_on "Apply filters"
-    expect(page).to have_selector(".report-section", text: "0")
+  def then_i_cannot_see_the_content_item_i_have_been_allocated
+    expect(@audit_report_page).to have_report_section(text: '0')
```

Comment:
`expect(@audit_report_page).to have_content_item_count(0)` ?

Reply:
As discussed have gone for `have_content_item_count(text: 0)